suites/apis/drsEndpointsTest.js

```diff
@@ -73,11 +73,18 @@ Scenario('get drs invalid access id @drs', async ({ drs, fence }) => {
   await fence.ask.responsesEqual(signedUrlRes, drs.props.resInvalidFileProtocol);
 });
 
-Scenario('get drs presigned-url no auth header @drs', async ({ drs, fence }) => {
+// New behavior where GA4GH API responds with a 401 with no access token
+Scenario('get drs presigned-url no auth header @drsNew', async ({ drs, fence }) => {
   const signedUrlRes = await drs.do.getDrsSignedUrlWithoutHeader(files.allowed);
   fence.ask.responsesEqual(signedUrlRes, drs.props.noAccessToken);
 });
 
+// Old behavior where GA4GH API responded with a 403 with no access token
+Scenario('get drs presigned-url no auth header @drs', async ({ drs, fence }) => {
```

Comment:
i don't think this will work as-is because we want to disable the old test when running the new test. Right now if you use `@drsNew` you will run both the old and the new test

Reply:
#699